keps/sig-network/2091-cluster-network-policy/README.md

```diff
@@ -139,15 +151,18 @@ Cluster Administrator. It is not intended to solve:
 
 In order to achieve the two primary broad use cases for a cluster admin to
 secure K8s clusters, we propose to introduce the following resource
-under `netpol.networking.k8s.io` API group:
+under `policy.networking.k8s.io` API group:
 - ClusterNetworkPolicy
 
 ### ClusterNetworkPolicy resource
 
-A ClusterNetworkPolicy (CNP) resource will help the administrators set strict
-security rules for the cluster, i.e. a developer CANNOT override these rules
-by creating NetworkPolicies that applies to the same workloads as the
+A ClusterNetworkPolicy (CNP) resource will help the administrators:
+1. Set strict security rules for the cluster, i.e. a developer CANNOT override
+these rules by creating NetworkPolicies that applies to the same workloads as the
 ClusterNetworkPolicy does.
+2. Set baseline security rules that describes the default connectivity verdict
+for cluster workloads, which CAN be overriden by developer NetworkPolicies if
+needed.
```

Comment:
nit: "that describes the default connectivity verdict"  -> "that describe default connectivity for cluster workloads"